syllabus/3-Blockchain/1-Overview_of_Blockchains_slides.md

```diff
@@ -6,6 +6,15 @@ duration: 30 - 45 minutes
 
 # Overview of Blockchains
 
+TODO Joshy
+
+Braindump for the structure of this lesson for HK:
+
+- Shared Story Telling
+- Digital Services
+- Broad def of smart contract
+- State Machines
```

Comment:
Is this pending for now? Or this will take place on another PR?

Reply:
Yeah, I'll do the slide content in a follow up so we can get the schedule on main.